salt/states/pkgrepo.py

```diff
@@ -412,14 +411,14 @@ def managed(name, ppa=None, **kwargs):
                         salt.utils.pkg.deb.combine_comments(kwargs['comments'])
                     if pre_comments != post_comments:
                         break
-            elif kwarg == 'comments' and os_family == 'redhat':
+            elif kwarg == 'comments' and __grains__['os_family'] == 'RedHat':
                 precomments = salt.utils.pkg.rpm.combine_comments(pre[kwarg])
                 kwargcomments = salt.utils.pkg.rpm.combine_comments(
                         sanitizedkwargs[kwarg])
                 if precomments != kwargcomments:
                     break
             else:
-                if os_family in ('redhat', 'suse') \
+                if __grains__['os_family'] in ('RedHat', 'Suse', 'SUSE') \
```

Comment:
Same here. It is "Suse".

Reply:
Ok. I'll change that, and hope I don't break anything.